vtex-apps/pixel-manager
# Remove offline events handling

#### What is the purpose of this pull request?
remove handling of events emitted when the user is offline and the localstorage logic.

#### What problem is this solving?
This won't be necessary anymore, since we'll move the offline handling of the events to the service worker with a workbox plugin.

#### How should this be manually tested?
access [this workspace](https://lucas2--storecomponents.myvtex.com/), disconnect your computer from the internet and reload the page. You'll be able to see the events in the indexed db, under the `workbox-background-sync` table. Then, you can reconnect your computer from the internet and send the sync signal with the following message `workbox-background-sync:workbox-google-analytics` and see the events being replayed.

#### Types of changes
